_api-reference/nodes-apis/nodes-stats.md

```diff
@@ -87,6 +88,16 @@ GET _nodes/stats/indices/docs,search
 ```
 {% include copy-curl.html %}
 
+You can also use specific `index_metric` values for the `caches` metric to specify which caches to return statistics for. 
```

Comment:
```suggestion
You can also use specific `index_metric` values in the `caches` metric to specify which caches to return statistics.
```

Reply:
```suggestion
You can also use specific `index_metric` values in the `caches` metric to specify which caches will return statistics. 
```
@Naarcha-AWS thoughts on changing it to above?

Reply:
This works for me.